marketdata/stream/flow.go

```diff
@@ -55,6 +55,7 @@ func (c *client) initialize(ctx context.Context) error {
 		if !isErrorRetriable(retryErr) {
 			return retryErr
 		}
+		c.logger.Infof("datav2stream: auth error: %s, will retry", retryErr)
```

Comment:
There is some logging on L40 that is similar to this one. Perhaps here you could omit the "will retry" part of the message and let the other one log that information (if it actually retries)
Oh and please fix the typo on L40 (retring)

Reply:
I fixed the typo. I find it useful to log it on both places.

Reply:
but here you might log `... will retry` even though there won't be any more retries

Reply:
You convinced me, I remove it.